package.json

```diff
@@ -45,44 +45,45 @@
 		"@eslint/js": "9.22.0",
 		"@release-it/conventional-changelog": "10.0.0",
 		"@types/eslint-plugin-markdown": "2.0.2",
-		"@types/node": "22.13.4",
+		"@types/node": "22.13.10",
 		"@typescript-eslint/rule-tester": "8.27.0",
-		"@vitest/coverage-v8": "3.0.5",
-		"@vitest/eslint-plugin": "1.1.31",
+		"@vitest/coverage-v8": "3.0.9",
+		"@vitest/eslint-plugin": "1.1.38",
 		"console-fail-test": "0.5.0",
-		"cspell": "8.17.3",
+		"create-typescript-app": "2.18.3",
+		"cspell": "8.17.5",
 		"eslint": "9.22.0",
 		"eslint-doc-generator": "2.1.0",
 		"eslint-plugin-eslint-plugin": "6.4.0",
-		"eslint-plugin-jsdoc": "50.6.3",
+		"eslint-plugin-jsdoc": "50.6.8",
 		"eslint-plugin-jsonc": "2.19.1",
 		"eslint-plugin-markdown": "5.1.0",
-		"eslint-plugin-n": "17.16.1",
-		"eslint-plugin-package-json": "0.26.0",
-		"eslint-plugin-perfectionist": "4.10.0",
+		"eslint-plugin-n": "17.16.2",
+		"eslint-plugin-package-json": "0.29.0",
+		"eslint-plugin-perfectionist": "4.10.1",
 		"eslint-plugin-regexp": "2.7.0",
 		"eslint-plugin-yml": "1.17.0",
 		"husky": "9.1.7",
 		"knip": "5.46.0",
 		"lint-staged": "15.5.0",
 		"markdownlint": "0.37.4",
 		"markdownlint-cli": "0.44.0",
-		"prettier": "3.5.1",
+		"prettier": "3.5.3",
 		"prettier-plugin-curly": "0.3.1",
-		"prettier-plugin-packagejson": "2.5.8",
+		"prettier-plugin-packagejson": "2.5.10",
 		"prettier-plugin-sh": "0.15.0",
 		"release-it": "18.1.2",
 		"sentences-per-line": "0.3.0",
 		"tsup": "8.4.0",
 		"typescript": "5.8.2",
 		"typescript-eslint": "8.27.0",
-		"vitest": "3.0.5"
+		"vitest": "3.0.9"
 	},
 	"peerDependencies": {
-		"@typescript-eslint/parser": ">=8",
```

Comment:
Haha, this is getting removed by `remove-dependencies`. I don't think `peerDependencies` should be.